src/libfsm/gen.c

```diff
@@ -208,8 +213,9 @@ gen_init_outer(struct fsm *fsm, size_t max_length,
 
 	assert(fsm_all(fsm, fsm_isdfa)); /* DFA-only */
 
-	assert(!randomized);		 /* not yet supported */
-	(void)seed;
+	if (randomized) {
+		srand(seed);
+	}
```

Comment:
Not sure about calling `srand()` in library code, because of its state being side-effect to the caller's own uses. I don't want to include our own prng. What do you think of moving the `srand()` invocation to the caller?

Reply:
Good point. Should the interface just have `bool randomized` (or `int`, since nothing else in that header uses `bool`), note that it uses `rand` specifically, and encourage the caller to seed with `srand`? It doesn't need to be a _good_ PRNG, just something that has more variation than always choosing the first printable character.

Reply:
Updated in cf8fc651.